scripts/wrappers/status.py

```diff
@@ -92,19 +125,21 @@ def get_status(available_addons, isReady):
 
     # initiate the parser with a description
     parser = argparse.ArgumentParser(description='Microk8s cluster status check.', prog='microk8s.status')
-    parser.add_argument("-o", "--output", help="print cluster and addon status, output can be in yaml or console",
-                        default="console", choices={"console", "yaml"})
-    parser.add_argument("-w", "--wait-ready", help="wait until the cluster is in ready state", nargs='?', const=True,
-                        type=bool)
+    parser.add_argument("--format", help="print cluster and addon status, output can be in yaml, pretty or short",
+                        default="short", choices={"pretty", "yaml", "short"})
+    parser.add_argument("-w", "--wait-ready", action='store_true', help="wait until the cluster is in ready state")
     parser.add_argument("-t", "--timeout",
                         help="specify a timeout in seconds when waiting for the cluster to be ready.", type=int,
                         default=0)
     parser.add_argument("-a", "--addon", help="check the status of an addon.", default="all")
+    parser.add_argument("--yaml", action='store_true', help="DEPRECATED, use '--format yaml' instead")
```

Comment:
This broke commandline compatibility by switching `-o yaml` -> `--yaml`. Since this was backported to all snap versions, I will just update Cilium make targets to use `--yaml`.

Reply:
I am sorry this broke you @joestringer. I do not think this change was backported, eg https://github.com/ubuntu/microk8s/blame/1.17/microk8s-resources/wrappers/microk8s-status.wrapper#L59 Perhaps there is something else we did and broke the integration?

Reply:
Hmm, weird, I thought I saw the change being applied to other versions but I guess I just got mixed up with another issue/PR.

Looking at my local setup, I'm tracking `latest/stable` on my machine so I would have picked up this change along with k8s 1.18 upgrade. Either way I think this will only affect a couple of Cilium developers and we're likely to stick to the latest version of microk8s anyway. The workaround on Cilium side is simple enough.